README.md

```diff
@@ -25,19 +25,47 @@ more about [automatic PHP error reporting with Sentry](https://sentry.io/for/php
 - Send customized diagnostic data
 - Process and sanitize data before sending it over the network
 
+## Install
+
+To install the SDK you will need to be using [Composer]([https://getcomposer.org/)
+in your project. To install it run:
+
+```bash
+curl -sS https://getcomposer.org/installer | php
+```
```

Comment:
It's common to have this set of instructions to install Composer on READMEs, but if you prefer to just link to the docs I will do it.